zip-0032.rst

```diff
@@ -713,6 +623,25 @@ spending keys, we expect that most wallets will only expose the regular Orchard
 users [#protocol-orchardspendingkeyencoding]_.
 
 
+Values reserved due to previous specification for Sprout
+========================================================
+
+The following values were previously used in the specification of hierarchical derivation
+for Sprout, and therefore SHOULD NOT be used in future Zcash-related specifications:
+
+* the BLAKE2b-256 personalizations
+
+  * :math:`\texttt{“ZcashIP32_Sprout”}`, formerly specified for derivation of the master
+    key of the Sprout tree;
```

Comment:
This is a BLAKE2b-512 personalization, not BLAKE2b-256. Fixed in 8e2215c577dae83f345118b77b864a7cb4a264a5.